conf/application.conf

```
@@ -170,6 +170,10 @@ features {
   # The only valid item value is currently "ConnectomeView":
   optInTabs = ["ConnectomeView"]
   openIdConnectEnabled = false
+  # If true, users can register with email/pw if there is exactly one orga.
+  # If false, they need an invite or SSO.
+  # Has no effect if isWkorgInstance is true.
+  registerToDefaultOrgaEnabled = true
```

```suggestion
  registerEnabled = true
```

why "toDefaultOrga"?